pkg/persistence/config/loader/config_loader_test.go

```diff
@@ -1225,6 +1225,62 @@ configs:
 `,
 			wantErrorsContain: []string{"unknown API: segment"},
 		},
+		{
```

Comment:
Is it possible to put this in a separate test? Maybe even in a separate file? 
This table taste is huge.

Reply:
The test file is 1k5 lines of code. If we want to do it, I can but in a separate PR.

Reply:
We discussed and the proper refactoring is going to be a separate task